code/WorkInProgress/Ported/policetape.dm

```diff
@@ -144,7 +144,7 @@
 		for(var/atom/A in T) //Check to see if there's anything solid on the tape's turf (it's possible to build on it)
 			if(A.density)
 				return
-		M:loc = T
+		M:forceMove(T)
```

Comment:
Mentally I treat those two as:
```verbname(X as Y)```
and
```input(stuff) as X```
I don't really consider them the same (despite the same keyword, ```as```)